BioArchLinux/spades/PKGBUILD

```diff
@@ -9,84 +7,34 @@ pkgrel=1
 pkgdesc='SPAdes Genome Assembler'
 url='http://cab.spbu.ru/software/spades/'
 arch=('x86_64')
-license=('GPLv2')
-depends=('python')
+license=('GPL2')
+depends=(
+  python
+)
 makedepends=(
-    'gcc'
-    'cmake'
-    'make'
-    'pkgconf'
-    'zlib'
-    'bzip2'
-    'bash'
-    'openmp'
+  bzip2
+  cmake
+  openmp
+  zlib
 )
-checkdepends=('python')
+options=("!lto")
```

Comment:
I don't know this part. Why it influences the build process ?

Reply:
Report it to upstream:
https://github.com/ablab/spades/issues/1007

Reply:
The author suggested us just do not build with LTO.
I think we can merge this PR.

Reply:
I think you can merge by yourself

Reply:
OK.